sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -279,8 +283,13 @@ export class AssessmentController {
           const updated =
             await this.studentAssessmentService.updateAssessmentWrapUpData(
               job.variables.assessmentId,
+              isOnlyAssessmentStatusUpdate,
```

Comment:
Pretty, pretty minor 😄 
While reading, it seems more natural to have it called `isAssessmentStatusUpdateOnly`.